src/test/java/tatracker/logic/commands/student/EditStudentCommandTest.java

```diff
@@ -1,177 +1,204 @@
-    //
-    //     String expectedMessage = String.format(EditStudentCommand.MESSAGE_EDIT_STUDENT_SUCCESS, editedStudent);
-    //
-    //     Model expectedModel = new ModelManager(new TaTracker(model.getTaTracker()), new UserPrefs());
-    //     expectedModel.setStudent(model.getFilteredStudentList().get(0), editedStudent);
-    //
-    //     assertCommandSuccess(editStudentCommand, model, expectedMessage, expectedModel);
-    // }
-
-    // @Test
-    // public void execute_someFieldsSpecifiedUnfilteredList_success() {
-    //     Index indexLastStudent = Index.fromOneBased(model.getFilteredStudentList().size());
-    //     Student lastStudent = model.getFilteredStudentList().get(indexLastStudent.getZeroBased());
-    //
-    //     StudentBuilder studentInList = new StudentBuilder(lastStudent);
-    //     Student editedStudent = studentInList.withName(VALID_NAME_BOB).withPhone(VALID_PHONE_BOB)
-    //             .withTags(VALID_TAG_HUSBAND).build();
-    //
-    //     EditStudentDescriptor descriptor = new EditStudentDescriptorBuilder().withName(VALID_NAME_BOB)
-    //             .withPhone(VALID_PHONE_BOB).withTags(VALID_TAG_HUSBAND).build();
-    //     EditStudentCommand editStudentCommand = new EditStudentCommand(indexLastStudent, descriptor);
-    //
-    //     String expectedMessage = String.format(EditStudentCommand.MESSAGE_EDIT_STUDENT_SUCCESS, editedStudent);
-    //
-    //     Model expectedModel = new ModelManager(new TaTracker(model.getTaTracker()), new UserPrefs());
-    //     expectedModel.setStudent(lastStudent, editedStudent);
-    //
-    //     assertCommandSuccess(editStudentCommand, model, expectedMessage, expectedModel);
-    // }
-
-    // @Test
-    // public void execute_noFieldSpecifiedUnfilteredList_success() {
-    //     EditStudentCommand editStudentCommand = new EditStudentCommand(INDEX_FIRST_STUDENT,
-    //             new EditStudentDescriptor());
-    //     Student editedStudent = model.getFilteredStudentList().get(INDEX_FIRST_STUDENT.getZeroBased());
-    //
-    //     String expectedMessage = String.format(EditStudentCommand.MESSAGE_EDIT_STUDENT_SUCCESS, editedStudent);
-    //
-    //     Model expectedModel = new ModelManager(new TaTracker(model.getTaTracker()), new UserPrefs());
-    //
-    //     assertCommandSuccess(editStudentCommand, model, expectedMessage, expectedModel);
-    // }
-
-    /*
+    private Model model = new ModelManager(getTypicalTaTrackerWithStudents(), new UserPrefs());
+
+    @Test
+    public void execute_allFieldsSpecifiedUnfilteredList_success() {
+        Student editedStudent = new StudentBuilder().build();
+        EditStudentDescriptor descriptor = new EditStudentDescriptorBuilder(editedStudent).build();
+        EditStudentCommand editStudentCommand = new EditStudentCommand(MATRIC_FIRST_STUDENT,
+                getTypicalModule().getIdentifier(), getTypicalGroup().getIdentifier(), descriptor);
```

Comment:
Maybe can store the group and module code as local variables, then won't be so long.

outside test: public String groupCode = getTypicalModule.getIdentifier(), etc.